test/resource/igniter_test.exs

```diff
@@ -111,4 +111,99 @@ defmodule Ash.Resource.IgniterTest do
       |> assert_unchanged()
     end
   end
+
+  describe "add_new_code_interface" do
+    setup do
+      igniter =
+        test_project()
+        |> Map.put(:args, %Igniter.Mix.Task.Args{
+          positional: %{
+            resource: "MyApp.Accounts.User"
+          },
+          options: [
+            domain: "MyApp.Accounts",
+            default_actions: ~w[create read update destroy],
+            uuid_primary_key: "id",
+            attribute: ["name:string:required:public"],
+            relationship: [],
+            extend: []
+          ]
+        })
+        |> Mix.Tasks.Ash.Gen.Resource.igniter()
```

Comment:
What we'd do here is:

```elixir
mix igniter.compose_task("ash.gen.resource", ["--cli", "flags", ...])
```

Reply:
Sorry, I said that weird. Just `Igniter.compose_task`, not `mix igniter.compose_task(...)`